lib/cursor.js

```diff
@@ -1000,6 +1018,12 @@ Cursor.prototype.transformStream = function(options) {
  * @return {Promise} returns Promise if no callback passed
  */
 Cursor.prototype.explain = function(callback) {
+  if (this.operation) {
+    this.operation.options.explain = true;
+    executeOperation(this.s.topology, this.operation, callback);
```

Comment:
should this not be `return executeOperation(...)` for the non-callback version?

Reply:
yep, thanks for the nice catch, fixed